tasks/copy_to_glacier/copy_to_glacier.py

```diff
@@ -101,13 +103,20 @@ def task(event: Dict[str, Union[List[str], Dict]], context: object) -> Dict[str,
     #      - collection configuration
     #      - default value in buckets
     try:
-        default_bucket = os.environ.get('ORCA_DEFAULT_BUCKET', None)
-        if default_bucket is None or len(default_bucket) == 0:
-            raise KeyError('ORCA_DEFAULT_BUCKET environment variable is not set.')
+        default_bucket = config.get(CONFIG_ORCA_DEFAULT_BUCKET_OVERRIDE_KEY)
     except KeyError:
         # TODO: Change this to a logging statement
-        print('ORCA_DEFAULT_BUCKET environment variable is not set.')
-        raise
+        print(f"{CONFIG_ORCA_DEFAULT_BUCKET_OVERRIDE_KEY} is not set. Using ORCA_DEFAULT_BUCKET environment value.")
```

Comment:
I am guesing this will need to be changed to logger.info later once orca-230 is merged.